src/scripting/lua_map.cc

```diff
@@ -3847,7 +3889,7 @@ int LuaWorkerDescription::get_employers(lua_State* L) {
 /* RST
    .. attribute:: buildable
 
-      (RO) returns true if this worker is buildable
+      (RO) Returns true if this worker is buildable.
```

Comment:
Not a function, so no "returns". Also type markup.

Reply:
There are some occurences like this. If we are strict in this way we have to remove also all markup `:returns: …` for attributes.

Reply:
Okay let's keep it like this for now